app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from os import rename, mkdir, remove, path, getcwd
+
+
+def move_file(command: str) -> None:
+    if command[-1] != "/" and len(command.split()) == 3:
```

Comment:
Do not check the presence of directory delimiters or use them in your logic, your code should be platform-agnostic. Use `os` module instead.